### PR TITLE
GROUNDWORK-1955: Add golangci-lint + README improvements

### PR DESCRIPTION
**Pull request by VladislavSenkevich**

### CODE REVIEW

File: services/services.go [hunk 1]
```diff
@@ -125,8 +125,7 @@ type Controllers interface {
 type payloadType byte
 
 const (
-	typeUndefined payloadType = iota
-	typeEvents
+	typeEvents payloadType = iota
```
Comment (ymkins) [inline services/services.go:128]: to prevent data migration should be like:
```
_ payloadType = iota
typeEvents
```

File: logzer/logzer_test.go [hunk 2]
```diff
@@ -63,8 +63,8 @@ func TestLogFilter(t *testing.T) {
 	log.Info().
 		Str("password", "PASSword").
 		Dict("dict", zerolog.Dict().Str("token", "TOKen")).
-		RawJSON("payload1", []byte(payload1)).
-		RawJSON("payload2", []byte(payload2)).
+		RawJSON("payload1", (payload1)).
+		RawJSON("payload2", (payload2)).
```
Comment (ymkins) [inline logzer/logzer_test.go:67]: it looks like parentheses are not needed

File: connectors/nsca-connector/nsca/nsca.go [hunk 3]
```diff
@@ -13,9 +13,9 @@ import (
 	"strings"
 	"time"
 
+	"github.com/gwos/tcg/connectors/nsca-connector/nsca/tools"
 	"github.com/rs/zerolog"
 	"github.com/rs/zerolog/log"
-	"github.com/tubemogul/nscatools"
```
Comment (ymkins) [inline connectors/nsca-connector/nsca/nsca.go:18]: lets discuss it

File: logzer/logzer_test.go (hunk 2 above)
Comment (VladislavSenkevich) [reply logzer/logzer_test.go]: Done ✓

**Comment by VladislavSenkevich** [comment]
- Revert nsca dep
- Revert gotocjson + exclude from linter
- Add TODO for enabledFilters

File: connectors/elastic-connector/clients/queryTranslator.go [hunk 4]
```diff
@@ -57,35 +58,26 @@ func filtersToESQueries(filters []KFilter, negate bool) []interface{} {
...
-	var cleanedFilters []interface{}
+	var cleanedFilters = make([]interface{}, 0, len(translatedFilters))
 	for _, filter := range translatedFilters {
 		cleanedFilters = append(cleanedFilters, cleanFilter(filter))
 	}
 
 	return cleanedFilters
 }
 
-func isFilterDisabled(filter KFilter) bool {
```
Comment (ymkins) [inline connectors/elastic-connector/clients/queryTranslator.go:82]: comment and mark TODO

**Review by ymkins — APPROVED** [review]
LGTM

**Comment by ymkins** [comment]
For today there are issues with running `golangci-lint` in CI
caused by https://github.com/golangci/golangci-lint/issues/2649.
Suggest to split this PR into two parts: 
* codebase changes are good for QA and merging
* linter configuration for automate run in CI should be disabled until issues resolved in upstream
Running linter in local env is good enough for today.

**Pull request merged**